gpio/gpioutil/debounce_test.go

```diff
@@ -84,6 +86,57 @@ func TestDebounce_WaitForEdge_Got(t *testing.T) {
 	}
 }
 
+func TestDebounce_WaitForEdge_Noise_NoEdge(t *testing.T) {
+	fakeClock := clockwork.NewFakeClock()
+	f := gpiotest.Pin{
+		Clock:     fakeClock,
+		EdgesChan: make(chan gpio.Level, 1),
+	}
+	p, err := Debounce(&f, time.Second, 0, gpio.BothEdges)
+	p.(*debounced).clock = fakeClock
+	f.Out(gpio.Low)
+	if err != nil {
```

Comment:
Move it to line 96? Otherwise the code would crash anyway, there's no chance this line ever runs.

Reply:
Must have missed that in one of the refactors.